### PR TITLE
Setup files of the Hello Microverse Project

### PR DESCRIPTION
**Pull request by jrraul4**
-Add the linters
-Add the html file
-Add the style.css file
-Edit the README file
-Fixing errors
-Solicitate the pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,214 @@
...
+
+Coming soon
+
+### Deployment
+
+You can deploy this project using:
+
+https://github.com/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (muneebulrehman) [inline README.md]: Kindly remove this part of your README file as it is not relevant to this project and to keep the README file clean.

File: README.md [hunk 2]
```diff
@@ -1 +1,214 @@
...
+
+I would like to thank to Microverse and to my coding partners for help me in this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (muneebulrehman) [inline README.md:199]: The link to the MIT license is broken. Kindly fix this issue.

File: README.md [hunk 3]
```diff
@@ -1 +1,214 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (muneebulrehman) [inline README.md]: Kindly remove this part from your README file. This text is provided for your guidance in the template and should not be included in the README file.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @jrraul4 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly give a proper title to your PR. Currently, your PR title is not relevant to the changes/new features you have implemented in this PR. Kindly do so before submitting for a re-review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving meaningful commit messages is important. I would recommend giving proper meaningful commit messages in the future. Not like **first review**, **second review**, etc.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link ref="stylesheet" href="./style.css">
```
Comment (muneebulrehman) [inline index.html]: I believe you have misspelled `ref` here. Instead it should be `rel`. This error is causing your CSS file not to load. Kindly fix this issue.

**Review by rica213 — APPROVED** [review]
# 🟢 STATUS: APPROVED
Hi @jrraul4 
My name is Larissa. I am your code reviewer  🥰

Thank you for implementing the required changes asked by the previous reviewer ❤️
Your readme file is clean and professional, I love it ❤️❤️ Please keep up the good work.
Your project is complete!  Let's merge it 🛸 🛸 
Congratulations! 🎉 



## Highlights

- [ ] No Linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jrraul4** [comment]
thanks to you

**Pull request merged**